Resources/js/router.js

```diff
@@ -263,8 +281,8 @@ class Router {
             url = route.requirements["_scheme"] + "://" + (host || this.getHost()) + url;
         } else if ("undefined" !== typeof route.schemes && "undefined" !== typeof route.schemes[0] && this.getScheme() !== route.schemes[0]) {
             url = route.schemes[0] + "://" + (host || this.getHost()) + url;
-        } else if (host && this.getHost() !== host) {
-            url = this.getScheme() + "://" + host + url;
+        } else if (host && this.getHost() !== host + ('' === portextension ? '' : ':' + portextension)) {
```

Comment:
this can then just be like here:

https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/pull/185/files#diff-8e1e018bfe2bd15df10d1864a922ff0cR257

Reply:
This part of code has been changed, because I remove the colon `:` in the `portextension` variable.
Like I said before, the `portextension` variable should only contain an int value with the port number